Crawler/PartOfSpeechTagger/PosTagger.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Crawler.ExtensionMethods;
+using Crawler.LexicalAnalyzer;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using Crawler.Configs;
+using Microsoft.Extensions.Options;
+
+namespace Crawler.PartOfSpeechTagger
+{
+	public class PosTagger : IPosTagger
```

Comment:
Rename this class the NodeJSPosTagger or somthing alike

Reply:
I am not really sure it's that important to indicate in the class name that it based on node js.